bindata/v4.1.0/aws-pod-identity-webhook/deployment.yaml

```diff
@@ -50,11 +50,11 @@ spec:
         node-role.kubernetes.io/master: ""
       priorityClassName: system-cluster-critical
       serviceAccountName: pod-identity-webhook
+      hostUsers: false
       securityContext:
         runAsNonRoot: true
         seccompProfile:
           type: RuntimeDefault
-        readOnlyRootFilesystem: true
```

Comment:
This is actually not supposed to be here, it's set on the container level already.